dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/NonLinear/FixedBooleanDelay.mo

```diff
@@ -0,0 +1,37 @@
+within Dynawo.NonElectrical.Blocks.NonLinear;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+block FixedBooleanDelay "Delay block with fixed delay time for boolean input"
+
+  import Modelica;
+  import Dynawo;
+
+  extends Modelica.Blocks.Interfaces.BooleanSISO(y(start = yStart));
+
+  parameter Dynawo.Types.Time delayTime "Delay time of output with respect to input signal";
+  parameter Boolean yStart "Initial value of output";
```

Comment:
Sometimes yStart, sometimes y_start it seems. I think we should choose once and for all. We could choose Y0 to be consistent with the dynawo coding style